README.md

```diff
@@ -1,112 +1,36 @@
-
-This extension can be installed in either one (or many) of:
-
-* [VS Code][code] or [Codium][codium]/Code-OSS, or
-* a [Theia][theia] application such as [Blueprint][blueprint].
-
-A nearby [companion extension][vscode-trace-extension] installation renders a `Trace Server`
-[status bar item][item]. A note:
-
-* [Theia Blueprint][blueprint] extracts installed extensions under `/tmp/vscode-unpacked/`.
-* Reinstalling an amended extension with the same version requires removing that extraction.
-* This is necessary beside uninstalling the extension from the Theia UI, that is.
-* Without this manual extension directory removal, Blueprint won't use the amended version.
-* Stepping the extension version upon an amend or update shouldn't trigger that issue.
 
 ## Configuration
 
-* Under the `Trace Server` preference settings, the trace server `path` can be entered.
-  * Otherwise, the default `/usr/bin/tracecompass-server` is assumed locally.
-* Command-line `arguments` can be optionally set, [any of these if Incubator][server] server.
-  * Multiple arguments have to be separated by a space character each.
-
-## Usage
-
-1. Use the `Trace Server: start (if stopped)` command to launch the trace server instance.
-1. The latter should be made of two related processes; `grep` for `tracecompass` or the like.
-1. Use the `Trace Server: stop or reset` command to kill both processes, stopping the server.
-1. Alternatively, exiting the application should automatically stop the started server if any.
-1. Now, `Trace Server: start (if stopped)` only starts the server if known as currently stopped.
-
-The extension checks for proper server startup/stopping; `ps` or [TSP][tsp] can be used alongside.
-
-* `Trace Server: stop or reset` resets the known `pid` to none, if stopped outside of the extension.
-* `Trace Server: stop or reset` used upon no previously started server (known `pid`) does nothing.
+The following preference settings can be used, under `Trace Server`.
 
-## Debugging
+* trace server `path` : enter the absolute path and filename of your trace server
+  * default value: `/usr/bin/tracecompass-server`
+* Command-line `arguments`: Enter any CLI arguments you want passed to your trace server. See [here][server] for valid options for the incubator version of the Trace Compass server
+  * Multiple arguments have to be separated by a space character
 
-* One may launch the extension using `Run Extension`, to debug it with breakpoints, as usual.
-* The same can be done for tests, launching `Extension Tests` to debug them.
-* The enabled breakpoints get bound only upon exercising the extension.
+## Usage
 
-## Development
+### Starting the Trace Server
 
-The usual [Prettier][prettier] and [ESLint][eslint] combo in VS Code or Codium OSS is used.
+Use the `Trace Server: start (if stopped)` command to launch the trace server instance. The latter should be made of two related processes; `grep` for `tracecompass` or the like.
```

Comment:
This for manually starting. There is also the automatic start upon open of a new trace.

Reply:
I have updated the README to take this information into account. Knowing that this automatic server start upon a trace being opened exist, has changed the focus a bit for me, so I have made a wider update than requested. Please let me know if you have further suggestions.